Makefile.in

```diff
@@ -341,7 +341,9 @@ OPT_CUDA_LDFLAGS = @OPT_CUDA_LDFLAGS@
 OPT_CUPTI = @OPT_CUPTI@
 OPT_CUPTI_IFLAGS = @OPT_CUPTI_IFLAGS@
 OPT_CUPTI_LDFLAGS = @OPT_CUPTI_LDFLAGS@
+OPT_GOTCHA_LDFLAGS = @OPT_GOTCHA_LDFLAGS@
```

Comment:
The OPT_GOTCHA... flags in the OpenCL branch should disappear. The master branch and develop branch already have flags for GOTCHA. We should not have two distinct sets of GOTCHA flags for the same purpose.